src/Classes/ServiceAPI/MyRadio_User.php

```diff
@@ -1372,6 +1401,17 @@ public function getEditForm()
                         ['value' => 'o', 'text' => 'Other']
                     ]
                 ]))
+                ->addField(new MyRadioFormField('contract', MyRadioFormField::TYPE_CHECK, [
```

Comment:
Isn't this supposed to be an option only checkable by someone PC-y (i.e. they sign a physical piece of paper)?